### ISSUE TITLE
New WCAG Criteria - Focus Not Obscured (Enhanced) - AAA

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/onrr.gov-site#2940]
Some of our focus indicators need to be stronger - might consider a two-color focus indicator. We are missing some focus indicators but we already have issues for that

**Comment by Maroyafaied** [comment]
Let's do some manual testing on links, collections, buttons and summarize what we should be fixing or how we can be more consistent.

**Comment by greensomes** [comment]
The USWDS focus styles have resolved this.
